grass7/raster/r.green/r.green.install/r.green.install.html

```diff
@@ -1,20 +1,53 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
+<html>
+<head>
+<title>GRASS GIS manual: r.green.install</title>
+<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
+<link rel="stylesheet" href="grassdocs.css" type="text/css">
+</head>
+<body bgcolor="white">
+<div id="container">
+
+<a href="https://grass.osgeo.org/grass-stable/manuals/index.html"><img src="grass_logo.png" alt="GRASS logo"></a>
+<hr class="header">
+
+<h2>NAME</h2>
+
+<em>r.green.install</em> - Toolset to check that all the necessary Python libraries like scipy
+and numexpr are present in the python path. The module also check post-installation troubles
+that sometimes may occur.
+
+<h2>KEYWORDS</h2>
+<a href="https://grass.osgeo.org/grass-stable/manuals/raster.html">raster</a>,
+<a href="https://grass.osgeo.org/grass-stable/manuals/topic_biomass.html">biomass topic</a>
+
+<!-- meta page description: Computes the residual energy potential of different renewable energies like biomass or hydropower -->
 <h2>DESCRIPTION</h2>
 
-<em>r.green.install</em> checks that all the necessary python libraries like scipy and numexpr are present in the python path. The module also check post-installation troubles that sometimes may occur.
+The module installs the missing Python libraries for different operating systems and
+provide a general menu in the GRASS GUI.
 
 <h2>NOTES</h2>
 
+<em>r.green.install</em> checks that all the necessary Python libraries like scipy and numexpr
+are present in the python path. The module also check post-installation troubles that sometimes
+may occur.
 
-The module installs the missing python librariers for different operating systems and provide a general menu in the grass GUI.
-
-<h3>Microsoft Windows</h3><br><br> 
-You should install <a href="https://grass.osgeo.org/download/software/ms-windows/">GRASS</a> with admin rights, stand-alone installer or OSGeo4W package, in directory with full controll permissions, to be able to intall the r.green modules.
+<h3>Installation on Microsoft Windows</h3>
```

Comment:
not sure this sentence is needed as in winGRASS addons are installed in the addons folder:

C:\Users\youruser\AppData\Roaming\GRASS7\addons

Reply:
It was there, written by the original author(s) - no idea if to keep it - @zarch ?